signature/jws/types.go

```diff
@@ -0,0 +1,113 @@
+package jws
+
+import (
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/notaryproject/notation-core-go/signature"
+)
+
+const (
+	headerKeyAlg                  = "alg"
+	headerKeyCty                  = "cty"
+	headerKeyCrit                 = "crit"
+	headerKeyExpiry               = "io.cncf.notary.expiry"
+	headerKeySigningTime          = "io.cncf.notary.signingTime"
+	headerKeySigningScheme        = "io.cncf.notary.signingScheme"
+	headerKeyAuthenticSigningTime = "io.cncf.notary.authenticSigningTime"
+)
+
+var headerKeys = []string{
```

Comment:
Should we rename it to `knownHeaderKeys`?

Reply:
Updated.

Reply:
If a header key is in the code, it is known. It is not possible to have an unknown key in a known place.

Reply:
We can add comments to explicitly state they are known keys in terms of the spec.

Reply:
So, lets name it `headerKeys` with comments.
```
// headerKeys includes all known keys for JWS protected header
// [JWS envelope]: https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md#protected-headers
var headerKeys = []string{
```